packaging/Dockerfile.centos8-nmstate-dev

```diff
@@ -2,7 +2,7 @@
 # Fedora official repository
 # (https://hub.docker.com/r/fedora/systemd-systemd/).
 # It enables systemd to be operational.
-FROM centos:8
+FROM docker.io/library/centos:8
```

Comment:
I do not think this is a common practice.
When running the build, you can specify the repository AFAIK.

Reply:
When there was only docker and a single registry, it was not yet a problem. Therefore it is unlikely that it is a common practice. What is the problem with this?

Reply:
I see no specific problem, but I would like to check what the k8s projects are doing (or plan to) as well.

Reply:
Seems like some use the full path and some do not.
Seems ok to proceed.

Reply:
This change is now split in #682 to simplify this PR.